man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the network fixed amount of delay to all packets going out of the interface. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayJitterSec=</varname></term>
+        <listitem>
+          <para>Specifies the random delay to be added to <varname>NetworkDelaySec=</varname>. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLimitPacket=</varname></term>
+        <listitem>
+          <para>Specifies the fifo limit (packets). An unsigned integer ranges 0 to 4294967294. Defaults to 100.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLossQuota=</varname></term>
```

Comment:
As above, has nothing to do with delay. Also the term "quota" sounds wrong here, it's just a probability, not a limited resource of which you take a portion/quota. Just `Loss=` or `LossRandom=` sounds better to me.

Note however that there are other loss models in netem (`state`, `gemodel`). If you intend to implement support for those in the near future, you should consider how their config syntax would look like, in order to make things as consistent and extensible as possible.

Reply:
We don't use that term . we use `quota`

Reply:
I didn't realize systemd was also redefining English vocabulary words :)

Seriously, I really don't think "quota" is appropriate here. This loss percentage is the probability with which a queued packet is dropped (instead of being sent out).

Reply:
Hmm may be we can change here. Original patch was that. @yuwata?